pkg/metrics/predefined_charts.go

```diff
@@ -246,6 +246,22 @@ func netpolCharts(group string) []metricslatest.Chart {
 		}, group, "")...)
 }
 
+func ipsecStatusChart(group string) []metricslatest.Chart {
+	sectionName := "IPsec"
+	charts := chartVariantsFor(&metricslatest.Chart{
+		Type:          metricslatest.ChartTypeLine,
+		SectionName:   sectionName,
+		DashboardName: mainDashboard,
+		Title:         "IPSEC flows rate",
```

Comment:
```suggestion
		Title:         "IPsec flows rate",
```
i missed that one :-)